src/modules/control_allocator/module.yaml

```diff
@@ -396,6 +396,8 @@ parameters:
         CA_SP0_ANG${i}:
             description:
                 short: Angle for swash plate servo ${i}
+                long: |
+                  The angle is measured clockwise (as seen from top), with 0 pointing forwards (X axis).
```

Comment:
Could you explain why the number of swash plate servos can only be 3 or 4? why not 2?